doc/src/manual/linear-algebra.md

```diff
@@ -75,7 +77,8 @@ julia> B = [1.5 2 -4; 2 -1 -3; -4 -3 5]
  -4.0  -3.0   5.0
 
 julia> factorize(B)
-Base.LinAlg.BunchKaufman{Float64,Array{Float64,2}}([-1.64286 0.142857 -0.8; 2.0 -2.8 -0.6; -4.0 -3.0 5.0], [1, 2, 3], 'U', true, false, 0)
+Base.LinAlg.BunchKaufman{Float64,Array{Float64,2}}
```

Comment:
I think that is intentional. https://github.com/JuliaLang/julia/pull/22345/files#diff-eb0ca647960f94f028bd3a7b2f3486b8R93 @andreasnoack